README.md

```diff
@@ -22,6 +22,7 @@ https://groups.google.com/a/opencontainers.org/forum/#!forum/tob (tob@opencontai
 
 * [Digest](https://github.com/opencontainers/tob/blob/master/proposals/digest.md)
 * [Image Format Spec](https://github.com/opencontainers/tob/tree/master/proposals/image-format)
+* [Selinux](https://github.com/opencontainers/tob/blob/master/proposals/selinux.md)
```

Comment:
I'm fine with all lower case, but where we uppercase anything I think we should use "SELinux".